src/main/java/racingcar/model/car/Cars.java

```diff
@@ -6,71 +6,61 @@
 import java.util.List;
 import java.util.stream.Collectors;
 
-import racingcar.dto.CarDto;
-import racingcar.util.RandomUtil;
+import racingcar.util.MovableStrategy;
 
 public class Cars {
-	private static final int MIN_RANDOM_BOUND = 0;
-	private static final int MAX_RANDOM_BOUND = 9;
-	private static final int MIN_MOVABLE_VALUE = 4;
+    private final List<Car> cars = new ArrayList<Car>();
```

Comment:
```suggestion
    private final List<Car> cars = new ArrayList<>();
```

`ArrayList` 대신 `List` 참조 👍

Reply:
이 코멘트에 Suggested change가 있어요~